src/pages/locations/index.tsx

```diff
@@ -1,37 +1,27 @@
-import apiClient from '@/src/api/client';
-import { useEffect, useState } from 'react';
-import { Location } from '../../api/client/models/Location';
-import LocationTable from '../../components/LocationTable';
-import PageWrapper from '../../components/PageWrapper';
+import apiClient from "@/src/api/client";
+import { useEffect, useState } from "react";
+import { Location } from "../../api/client/models/Location";
+import LocationTable from "../../components/LocationTable";
+import PageWrapper from "../../components/PageWrapper";
 
 const LocationList = () => {
 	const [locations, setLocations] = useState<Location[] | undefined>(undefined);
 
 	const fetchLocations = () => {
-		apiClient.discoverCulturalData
-			.getLocations()
-			.then((res) => {
-				const locationsObject = res?.data?.locations;
-				setLocations(locationsObject || undefined);
-			})
-			.catch((error) => {
```

Comment:
Kannst du sagen, warum der catch hier überflüssig ist?

Reply:
Davor haben wir nur den Fehler über die `console` geloggt. Ich habe jetzt aber alle `console.log`-Statements entfernt (würde ich nur während lokaler Entwicklung erlauben, ansonsten ist die Gefahr zu hoch, dass sowas im Production Code landet). Von daher wäre da nur ein leeres `catch((error) => {})` übrig geblieben, das alle Fehler geschluckt/ignoriert hätte. Deswegen habe ich es lieber komplett entfernt. In Zukunft würde ich die ganzen API calls eh noch zu `async/await` mit `try/catch`-Blöcken migrieren.